src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/fetchMarkup/index.js

```diff
@@ -15,7 +13,7 @@ const fetchMarkup = async (url, assetId) => {
       headers: {
         'User-Agent': 'Simorgh/ws-web-rendering',
       },
-      timeout: SECONDARY_DATA_TIMEOUT,
+      signal: AbortSignal.timeout(SECONDARY_DATA_TIMEOUT),
```

Comment:
I haven't seen signal used before ! Why is it preferred over timeout?

Reply:
The native version of `fetch` doesn't have a built-in mechanism for `timeout`, so we need to use the native `AbortSignal` functionality provided to do a similar thing for us: https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal/timeout_static